tuna/miopen/driver/base.py

```diff
@@ -54,12 +61,40 @@ def __init__(self, line=None, db_obj=None):
       raise ValueError(
           "Error creating Driver. MIOpen Driver cmd line or db_obj required")
 
+  def parse_fdb_key(self, line):
+    """Overloaded method.Defined in conv&bn driver child class"""
+
+  def parse_row(self, db_obj):
+    """Overloaded method.Defined in conv&bn driver child class"""
+
+  def set_cmd(self, data_type):
+    """Overloaded method.Defined in conv&bn driver child class"""
+
+  def config_set_defaults(self):
+    """Overloaded method.Defined in conv&bn driver child class"""
+
+  @abstractmethod
+  def compose_weight_t(self):
+    """Overloaded method.Defined in conv&br driver child class"""
+
   @staticmethod
-  def get_common_cols():
+  def test_skip_arg(tok1):
+    """Overloaded method.Defined in conv&br driver child class"""
+
+  @staticmethod
+  def get_params(tok1):
+    """Overloaded method.Defined in conv&br driver child class"""
+
+  @staticmethod
+  def get_check_valid(tok1, tok2):
+    """Overloaded method.Defined in conv&br driver child class"""
+
```

Comment:
raise NotImplementedError("Not implemented")

Reply:
Good suggestion!